corehq/apps/app_manager/app_schemas/app_case_metadata.py

```diff
@@ -1,6 +1,22 @@
 from __future__ import absolute_import, unicode_literals
```

Comment:
nit: the name of this file seems redundant with the previous two levels in the hierarchy. What do you think of renaming to `case_metadata.py`?